test/framework/clustervalidator.go

```diff
@@ -18,14 +18,15 @@ import (
 )
 
 func validationsForExpectedObjects() []clusterf.StateValidation {
-	mediumRetier := retrier.NewWithMaxRetries(120, 5*time.Second)
-	longRetier := retrier.NewWithMaxRetries(120, 10*time.Second)
+	mediumRetier := retrier.New(10 * time.Minute)
+	longRetier := retrier.New(30 * time.Minute)
```

Comment:
Increasing these timeouts so that API tests can be less flaky. They seem to be taking a long time on Cloudstack especially from recent tests.